rpc/ub/plot.sh

```diff
@@ -128,21 +128,21 @@ gen_csv() {
     local VAR_OPT="$1"; shift
     local VALUES="$*"
 
-    local NR_MSGS=$(get_val nr_msgs $COMMON_OPTS)
+    local NR_MSGS=$(get_val nr_msgs "$COMMON_OPTS")
```

Comment:
Double quotes will merge multiple args into single arg. please avoid doing this.

Reply:
Done

Reply:
Done